Marlin/src/pins/stm32f1/pins_JGAURORA_A5S_A1.h

```diff
@@ -39,7 +39,7 @@
   #define STM32_XL_DENSITY
 #endif
 
-//#define MCU_STM32F103ZE // not yet required
+//#define MCU_STM32F103ZE                         // not yet required
```

Comment:
I think this change is here accidentally, unless this is an artifact of needing to merge new changes from bugfix-2.0.x

```suggestion
//#define MCU_STM32F103ZE // not yet required
```